src/pipecat/processors/audio_buffer_processor.py

```diff
@@ -0,0 +1,75 @@
+from pipecat.frames.frames import (AudioRawFrame, BotStartedSpeakingFrame,
```

Comment:
It'd be great if we could move this to `src/pipecat/processors/audio`.

Reply:
Done.